### DISCUSSION TITLE
Version 5.0

### DISCUSSION THREAD

**Discussion by Akascape** [Akascape/CTkDesigner-Support#17] [New Update]
## What's new in version 5.0?
![Screengif](https://github.com/Akascape/CTkDesigner-Support/assets/89206401/a2efbd88-eebf-4d34-aa36-d8c046584c93)
- New widget: ScrollableFrame
- Now you can add widgets in both normal and scrollable frame
- Acrylic theme for windows
- Alpha/transparency in widgets using pyinstyles
- Modified the app detail section where you can add title/icon in the app
- Fixed issues related to images #15 #14 #13 #12
- Updated ctkdlib

**Comment by teijiIshida** [comment]
Hello, 

I lock a CTkLabel. Is there anyway to unlock it?

Also, I found a bug. In the parameter box on top right. If I press Enter, it acts like I press Backspace.
![2024-06-06 14_05_54-](https://github.com/Akascape/CTkDesigner-Support/assets/33733459/5207eaa5-3b29-4385-b49d-eb9bb761a77e)

Another bug:
I have a CTkLabel with an image and no text. If I save json and then reload later, the image disappear and I can't select that label anymore. But if I have an image label with any text, then when I reload the json later, the image still disappear, but I can see the label text.

Feature request:
-When adjusting font size, there is no way to see the current size. Can you add a label to show the current font size? Sometimes I just want to see, so I can use the same size in another label in another page, for example.
![2024-06-06 19_24_33-Image Dialog](https://github.com/Akascape/CTkDesigner-Support/assets/33733459/7301b8a4-6d64-4735-a2eb-84b7f7d44931)
-Can you add more options for tooltips? For example, changing delay or transparency.
-Can you add an easy way to copy a widget from one page to another?

Would you prefer I create separate threads to ask questions like this or just post in this thread?

**Comment by Akascape** [comment]
@teijiIshida 

> I have a CTkLabel with an image and no text. If I save json and then reload later, the image disappear and I can't select that label anymore. But if I have an image label with any text, then when I reload the json later, the image still disappear, but I can see the label text.

Improvements done in v6.0
- Added unlock option
- Fixed the label image issue
- Added segmented button page navigation (serially-auto)
- change widget page option
- font scale label added
- delay for tooltip
- fixed enter button bug
- fixed spawn widgets overlapping with fixed widget 
- new document option
- quick preview option...

Give more relevent suggestions and I will try to add them :) 
Expect the update by tomorrow

**Reply by teijiIshida** [reply]
Perfect! Thank you.

I don't know if this is a bug or intentional, but I think I should mention it anyway.

I use textbox with pre-entered text inside. However, when I reload the json later, the text is no longer there. I could fix it in .py when I build my app, but during GUI development, I have to guess what the textbox is. A minor issue but still an issue nonetheless. I hope you can make the pre-entered texxt save in the json and load it whenever I open the template.

pre-entered text in textbox:
![1](https://github.com/Akascape/CTkDesigner-Support/assets/33733459/8e9f1e06-0930-48b3-a143-fbdd921003b2)

text gone when open template next time:
![2](https://github.com/Akascape/CTkDesigner-Support/assets/33733459/47baba39-102a-4b07-a6e9-9499684d56d3)

**Reply by Akascape** [reply]
The text is not saved in any textbox, entry, so you can better insert them after exporting the app with .insert. Even in the exported app it will not save those automatically, you have to manually insert them in code or save them somewhere. But I can try to add this feature separately.

**Reply by teijiIshida** [reply]
Got it.
